src/gordon_gcp/plugins/service/enricher.py

```diff
@@ -195,6 +197,27 @@ def _get_internal_ip(self, instance_data):
                 fqdn, external_ip, default_ttl, msg_logger)
         ]
 
+    async def _get_matching_records(self, instance_resource_url):
+        # Get the fqdn of the instance
+        instance_name = instance_resource_url.split('/')[-1]
+        fqdn = self._get_fqdn(instance_name)
+
+        # Get all Google DNS records for the managed zone
+        project = self.config.get('project')
```

Comment:
Are we sure this isn't required?  If it is, it should be validated in the builder, and accessed directly here.